clients/rust/src/lib.rs

```diff
@@ -1,4 +1,32 @@
 mod generated;
 
+use {
+    bytemuck_derive::{Pod, Zeroable},
+    solana_program::pubkey::Pubkey,
+    spl_pod::option::PodOption,
+};
+
 pub use generated::programs::SOL_STAKE_VIEW_PROGRAM_ID as ID;
 pub use generated::*;
+
+/// Helper struct to easily handle the return data created by the
+/// `GetStakeActivatingAndDeactivating` instruction.
+#[repr(C)]
+#[derive(Clone, Copy, Debug, PartialEq, Pod, Zeroable)]
+pub struct GetStakeActivatingAndDeactivatingReturnData {
+    pub delegated_vote: PodOption<Pubkey>,
+    pub effective: u64,
+    pub activating: u64,
+    pub deactivating: u64,
+}
+
+impl Default for GetStakeActivatingAndDeactivatingReturnData {
+    fn default() -> Self {
+        Self {
+            delegated_vote: None.try_into().unwrap(),
```

Comment:
super nit, completely optional: perhaps using `PodOption::from(Pubkey::NONE)` my look less intimidating.

Reply:
Yeah, I see what you mean.